src/main/java/com/bunq/sdk/http/ApiClient.java

```diff
@@ -316,8 +316,8 @@ private static String getResponseId(Response response) {
 
   /**
    */
-  private static void assertResponseSuccess(int responseCode, byte[] responseBodyBytes, String responseId) {
-    if (Pattern.matches(OK_STATUS_CODE_RANGE, responseId)) {
+  private static void assertResponseSuccess(Integer responseCode, byte[] responseBodyBytes, String responseId) {
```

Comment:
Is it possible here that the response code is null? (we had the situation before when the request doesn't reach the server (timeouts))